package.json

```diff
@@ -34,7 +34,9 @@
     "blueprint-markdown-renderer": "^1.0.0",
     "lodash": ">=3.10.1",
     "lodash-api-description": "0.0.2",
-    "media-typer": "^0.3.0"
+    "media-typer": "^0.3.0",
+    "minim": "^0.20.1",
```

Comment:
I'd update this to 0.20.2 as it includes the fix for serialising 0.6 arrays which likely happen during array serialisation.